assets/l10n/app_en.arb

```diff
@@ -27,28 +27,298 @@
+  "@composeBoxStreamContentHint": {
+    "description": "Hint text for content input when sending a message to a stream",
+    "placeholders": {
+      "stream": {"type": "String", "example": "stream name"},
+      "topic": {"type": "String", "example": "topic name"}
+    }
+  },
+  "composeBoxSendTooltip": "Send",
+  "@composeBoxSendTooltip":  {
+    "description": "Tooltip for send button in compose box."
+  },
+  "composeBoxUnknownStreamName": "(unknown stream)",
+  "@composeBoxUnknownStreamName": {
+    "description": "Replacement name for stream when it cannot be found in the store."
+  },
+  "composeBoxTopicHintText": "Topic",
+  "@composeBoxTopicHintText":  {
+    "description": "Hint text for topic input widget in compose box."
+  },
+  "composeBoxUploadingFilename": "Uploading {filename}...",
+  "@composeBoxUploadingFilename": {
+    "description": "Label in compose box showing the specified file is currently uploading.",
+    "placeholders": {
+      "filename": {"type": "String", "example": "file.txt"}
+    }
+  },
+  "contentValidationErrorTooLong": "Message length shouldn't be greater than 10000 characters.",
+  "@contentValidationErrorTooLong": {
+    "description": "Content validation error message when the message is too long."
+  },
+  "contentValidationErrorEmpty": "You have nothing to send!",
+  "@contentValidationErrorEmpty": {
+    "description": "Content validation error message when the message is empty."
+  },
+  "contentValidationErrorQuoteAndReplyInProgress": "Please wait for the quotation to complete.",
+  "@contentValidationErrorQuoteAndReplyInProgress": {
+    "description": "Content validation error message when a quotation has not completed yet."
+  },
+  "contentValidationErrorUploadInProgress": "Please wait for the upload to complete.",
+  "@contentValidationErrorUploadInProgress": {
+    "description": "Content validation error message when attachments have not finished uploading."
+  },
+  "dialogCancel": "Cancel",
+  "@dialogCancel": {
+    "description": "Button label in dialogs to cancel."
+  },
+  "dialogContinue": "Continue",
+  "@dialogContinue": {
+    "description": "Button label in dialogs to proceed."
+  },
+  "errorDialogContinue": "OK",
+  "@errorDialogContinue": {
+    "description": "Button label in error dialogs to acknowledge error."
+  },
+  "errorDialogTitle": "Error",
+  "@errorDialogTitle": {
+    "description": "Generic title for error dialog."
+  },
+  "lightboxCopyLinkTooltip": "Copy link",
+  "@lightboxCopyLinkTooltip": {
+    "description": "Tooltip in lightbox for the copy link action."
+  },
+  "loginPageTitle": "Log in",
+  "@loginPageTitle": {
+    "description": "Page title for login page."
+  },
+  "loginFormSubmitLabel": "Log in",
+  "@loginFormSubmitLabel": {
+    "description": "Button text to submit login credentials."
+  },
+  "loginAddAnAccount": "Add an account",
+  "@loginAddAnAccount": {
+    "description": "Page title for screen to add a Zulip account."
```

Comment:
I like the convention you've used elsewhere of having the name end in "page title" when it's the title of a page. Let's apply that consistently here, too.

Reply:
bump -- "page title" :-)